modules/Makefile.kubernetes

```diff
@@ -10,9 +10,10 @@ KUBERNETES_RESOURCE_VALIDATION ?= configmap
 export KUBERNETES_APP
 
 # Kubectl specific settings
+SSH_SOCK ?= ~/tmp/kubectl-$(CLUSTER_BASTION)
```

Comment:
I can use username, random value won't work since if we're running in a script like
```
make kubernetes:tunnel-up
make kubernetes:cmd1
make kubernetes:cmd2
make kubernetes:tunnel-down
```

the value would need to be the same across invocations.